recipes/tcb-span/all/conanfile.py

```diff
@@ -0,0 +1,31 @@
+from conans import ConanFile, tools
+import os
+import glob
```

Comment:
unused

```suggestion
```

Reply:
Thanks!  I  just copy&pasted that :)

BTW, I found a few days ago some package that has an assertion for conan to require a minimum compiler version.

I'll use it here (I hope to do it this evening).